animalfacts.py

```diff
@@ -2249,6 +2263,7 @@ def animalfactsbot(reddit):
     LIZARD_FACTS,
     LOBSTER_FACTS,
```

Comment:
Meerkats are social creatures as they live in groups called mobs, that can contain up to 40 members.